### PR TITLE
[AMS-817] Fix high CPU utilization

### PR DESCRIPTION
**Pull request by xakep139**

### CODE REVIEW

**Comment by denisivan0v** [comment]
In c7bc0dd the latest CI build of librdkafka with the fix of https://github.com/edenhill/librdkafka/issues/1384 was referenced.

Update to the next official release needed when it will become available.

**Comment by denisivan0v** [comment]
Custom manually created nupkg used in fd5812

File: src/VStore/Kafka/ConsumerWrapper.cs [hunk 1]
```diff
@@ -52,8 +52,10 @@ public virtual void Dispose()
                     { "queued.min.messages", 1000 },
 #if DEBUG
                     { "debug", "msg" },
+#else
+                    { "log.connection.close", "false" },
```
Comment (xakep139) [inline src/VStore/Kafka/ConsumerWrapper.cs]: https://github.com/edenhill/librdkafka/wiki/FAQ#why-am-i-seeing-receive-failed-disconnected

File: src/VStore/Kafka/EventSender.cs [hunk 2]
```diff
@@ -31,8 +31,12 @@ public EventSender(KafkaOptions kafkaOptions, ILogger<EventSender> logger)
                     { "socket.blocking.max.ms", 1 },
                     { "queue.buffering.max.ms", 5 },
                     { "queue.buffering.max.kbytes", 10240 },
+                    { "batch.num.messages", 100 },
+                    { "message.max.bytes", 10240 },
 #if DEBUG
                     { "debug", "msg" },
+#else
+                    { "log.connection.close", "false" },
```
Comment (xakep139) [inline src/VStore/Kafka/EventSender.cs]: https://github.com/edenhill/librdkafka/wiki/FAQ#why-am-i-seeing-receive-failed-disconnected

File: src/VStore.Host/VStore.Host.csproj [hunk 3]
```diff
@@ -23,14 +22,13 @@
...
   </ItemGroup>
-  
   <ItemGroup>
     <ProjectReference Include="..\VStore\VStore.csproj" />
   </ItemGroup>
-  
   <ItemGroup>
     <PackageReference Include="Autofac" Version="4.6.1" />
     <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="4.2.0" />
+    <PackageReference Include="librdkafka.redist" Version="0.11.1-a212ea2" />
```
Comment (denisivan0v) [inline src/VStore.Host/VStore.Host.csproj]: Please delete this package from `lib` folder

File: src/VStore/VStore.csproj [hunk 4]
```diff
@@ -9,10 +9,11 @@
     <RootNamespace>NuClear.VStore</RootNamespace>
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="librdkafka.redist" Version="0.11.1-RC1" />
```
Comment (xakep139) [inline src/VStore/VStore.csproj]: Main change

File: src/VStore.Host/VStore.Host.csproj [hunk 5]
```diff
@@ -7,10 +7,9 @@
     <PackageId>VStore.Host</PackageId>
     <!--<RuntimeIdentifier>osx.10.12-x64</RuntimeIdentifier>-->
     <RuntimeIdentifier>win10-x64</RuntimeIdentifier>
-    <ServerGarbageCollection>false</ServerGarbageCollection>
+    <ServerGarbageCollection>true</ServerGarbageCollection>
```
Comment (xakep139) [inline src/VStore.Host/VStore.Host.csproj]: Turning this back

File: src/VStore.Host/VStore.Host.csproj [hunk 6]
```diff
@@ -50,7 +47,6 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.FileExtensions" Version="1.1.2" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="1.1.2" />
     <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="1.1.2" />
-    <PackageReference Include="AWSSDK.S3" Version="3.3.10.1" />
```
Comment (xakep139) [inline src/VStore.Host/VStore.Host.csproj]: Doesn't needed here

File: src/VStore.Worker/VStore.Worker.csproj [hunk 7]
```diff
@@ -43,7 +43,6 @@
     <PackageReference Include="Polly" Version="5.3.1" />
     <PackageReference Include="Prometheus.Client.MetricServer" Version="1.1.2" />
     <PackageReference Include="System.Console" Version="4.3.0" />
-    <PackageReference Include="AWSSDK.S3" Version="3.3.10.1" />
```
Comment (xakep139) [inline src/VStore.Worker/VStore.Worker.csproj]: Doesn't needed here

File: src/VStore.Host/VStore.Host.csproj (hunk 3 above)
Comment (xakep139) [reply src/VStore.Host/VStore.Host.csproj]: Done in https://github.com/2gis/nuclear-vstore/pull/114/commits/d0417437d856d8153a47c1300293e501a56f883e

File: src/VStore.Host/VStore.Host.csproj (hunk 6 above)
Comment (denisivan0v) [reply src/VStore.Host/VStore.Host.csproj]: `AWSSDK.Extensions.NETCore.Setup` needed it when app is configuring, is't it?

File: src/VStore/VStore.csproj (hunk 4 above)
Comment (denisivan0v) [reply src/VStore/VStore.csproj]: So, this leads to transitive version updates, right?

File: src/VStore.Host/VStore.Host.csproj (hunk 6 above)
Comment (denisivan0v) [reply src/VStore.Host/VStore.Host.csproj]: Please test it more closely

**Comment by xakep139** [comment]
> `AWSSDK.Extensions.NETCore.Setup` needed it when app is configuring, is't it?

@denisivan0v no, just `AWSSDK.Core` as it's internal dependency. Besides, `AWSSDK.S3` package exists in VStore's project dependencies

File: src/VStore.Host/VStore.Host.csproj (hunk 6 above)
Comment (xakep139) [reply src/VStore.Host/VStore.Host.csproj]: No, just `AWSSDK.Core` as it's internal dependency. Besides, `AWSSDK.S3` package exists in VStore's project dependencies

File: src/VStore/VStore.csproj (hunk 4 above)
Comment (xakep139) [reply src/VStore/VStore.csproj]: That's right

**Pull request merged**